CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `__detailed_return` parameter added to `text2art` function
 - `__word2art` function modified
 - `sapce` bug fixed
+- `space` parameter added to `art` and `aprint` function
```

Comment:
I prefer:
```
- `space` parameter added to `art` function
- `space` parameter added to `aprint` function
```

Reply:
@codewithnick Please address this comment 💯